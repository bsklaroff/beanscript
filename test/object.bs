cat = {x: 5, y: 10}
horse = [-100, 99]
abc = {
  zebra: 23
  cat: cat
  horse: horse
  samecat: cat
  samehorse: horse
}
print(abc.zebra)
print(abc.cat.x)
print(abc.cat.y)
print(abc.samecat.x)
print(abc.samecat.y)
print(abc.horse[0])
print(abc.horse[1])
print(abc.samehorse[0])
print(abc.samehorse[1])
cat.x = 7
horse[0] = -98
abc.zebra = 47
print(abc.zebra)
print(abc.cat.x)
print(abc.cat.y)
print(abc.samecat.x)
print(abc.samecat.y)
print(abc.horse[0])
print(abc.horse[1])
print(abc.samehorse[0])
print(abc.samehorse[1])
