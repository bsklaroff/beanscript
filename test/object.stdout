23
5
10
5
10
-100
99
-100
99
47
7
10
7
10
-98
99
-98
99
